java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -827,6 +827,19 @@ public ColumnVector castTo(DType type) {
     return super.castTo(type);
   }
 
+  /**
+   * Create a new column that has data copied from the current column and a new bitmask copied from
+   * the given templateBitmask column.
+   *
+   * The caller is responsible for any data corruption that is make by calling to this function.
+   *
+   * @param templateBitmask a column from which the bitmask will be copied to the output column.
+   */
+  public ColumnVector replaceBitmask(ColumnView templateBitmask) {
```

Comment:
This should be a member function of `ColumnView` not `ColumnVector` if it's needed, but isn't this functionality already covered by `mergeAndSetValidity` in `ColumnView`?

Reply:
What??? Yes, `mergeAndSetValidity` can achieve the same result here.
So should we keep that new function? I think it may be useful somewhere else later, because `mergeAndSetValidity` only supports `OR` and `AND` the bitmask, not `REPLACE`.

Reply:
IIUC `mergeAndSetValidity` always replaces the bitmask.  The binary op is for how it combines the validity mask of the other input columns, but it shouldn't matter if you're only passing one column to use as an input for the validity mask.  cc: @mythrocks

Reply:
That's right. Sorry I missed it.
Alright, I'm going to remove this now.